OutSystems/outsystems-datagrid
# ROU-4329: Implement the ability to save/load layouts with frozen columns

This PR is for implement the ability to save/load layouts with frozen columns.

### What was done
* Implement getViewLayout and setViewLayout methods for ColumnFreeze feature
* Updated the getViewLayout and setViewLayout methods of View feature to set and save the freeze columns.

### Test Steps
1. Go to the test page
2. Freeze the columns
3. Click in "Save Layout"
4. Click in "Apply Layout"
5. Check if the columns are frozen.


### Checklist
